src/app/features/home/capture-tab/capture-item/capture-item.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, Input } from '@angular/core';
+import { UntilDestroy } from '@ngneat/until-destroy';
+import { BehaviorSubject } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { getOldProof } from '../../../../shared/services/repositories/proof/old-proof-adapter';
+import { Proof } from '../../../../shared/services/repositories/proof/proof';
+import { isNonNullable } from '../../../../utils/rx-operators/rx-operators';
+
+@UntilDestroy({ checkProperties: true })
+@Component({
+  selector: 'app-capture-item',
+  templateUrl: './capture-item.component.html',
+  styleUrls: ['./capture-item.component.scss'],
+})
+export class CaptureItemComponent {
+  @Input()
+  set item(value: CaptureItem) {
```

Comment:
What is the different between this approach (creating a BehaviorSubject in the component and pass the object from parent component through Input property) and directly use the Input property?

The CaptureItem from parent component is created from a subscribed observable, wouldn't it be propagated and updated automatically by Angular, or does it have something to do with avoiding additional re-rendering?